dataverse-webapp/src/main/webapp/filesFragment.xhtml

```diff
@@ -134,22 +134,15 @@
                 </div>
             </div>
 
-            <div class="row" jsf:rendered="#{datasetFilesTab.fileLabelSearchTerm.equals('') ?
+                <div class="row" jsf:rendered="#{datasetFilesTab.fileLabelSearchTerm.equals('') ?
```

Comment:
Can you point what changes are done in this file? Or is it indentation change only?

Reply:
Nothing was changed in this file. I had to merge it with develop because version on my branch was corrupted (I have branched off from develop when this file was corrupted on develop aswell). 
Github didn't merge it automatically so i had to resolve conflicts by hand and I must have added the identation by accident.
I guess this file could be omitted from merge altogether, but it must be in pull request for app to work on this branch.

Reply:
Can you double check before - after merge. I'm kind of worried about that.